mbed_flasher/erase.py

```diff
@@ -36,12 +36,15 @@ def __init__(self):
         logger = Logger('mbed-flasher')
         self.logger = logger.logger
 
-    def erase(self, target_id=None, no_reset=None, method=None):
+    def erase(self, target_id=None, no_reset=None, method=None,
```

Comment:
Just for the sake of API sanity.. Should there be just one function per erase method, then each will only expose relevant arguments

Reply:
The current approach enables the cli API and python API to be same.